routes/vetoes.js

```diff
@@ -181,26 +181,27 @@ router.post('/replaceMediator/:id', api.isNat, api.isNotSpectator, async (req, r
     let v = await vetoesService.query({ _id: req.params.id }, defaultPopulate);
     let currentMediators = [];
     v.mediations.forEach(mediation => {
-        if(mediation.id != req.body.mediationId){
-            currentMediators.push(mediation.mediator.id);
-        }
+        currentMediators.push(mediation.mediator.id);
     });
 
     const allUsers = await usersService.getAllMediators();
     for (let i = 0; i < allUsers.length; i++) {
         let user = allUsers[i];
         if (
-            user.modes.indexOf(v.mode) >= 0 &&
-            user.probation.indexOf(v.mode) < 0 &&
-            user.osuId != v.vetoer.osuId &&
-            user.osuId != v.beatmapMapperId
+            !currentMediators.includes(String(user._id))
```

Comment:
use user.id, don't need to stringify the _id

Reply:
seems like it's returning ``undefined`` though

Reply:
works fine for me? what exactly are you doing hmmm

Reply:
> use user.id, don't need to stringify the _id

well seems like you mean ``_id`` lol